client/packages/common/src/ui/components/buttons/standard/BaseButton.tsx

```diff
@@ -16,44 +16,47 @@ const translateColor = (theme: Theme, color?: string) => {
   }
 };
 
-export const StyledBaseButton = styled(MuiButton)(
-  ({ theme, color, variant }) => {
-    const getHoverBgColor = () =>
-      variant === 'contained'
-        ? theme.palette.background.white
-        : translateColor(theme, color);
+export const StyledBaseButton = styled(MuiButton)(({
```

Comment:
the change here is to include the style prop so that I can make a shorter button, as per the designs